internal/provider/model/job_definition/input_option/snowflake.go

```diff
@@ -0,0 +1,116 @@
+package input_options
+
+import (
+	"terraform-provider-trocco/internal/client/entity/job_definition/input_option"
+	input_options2 "terraform-provider-trocco/internal/client/parameter/job_definition/input_option"
```

Comment:
How about changing the alias name?
```suggestion
	parameter "terraform-provider-trocco/internal/client/parameter/job_definition/input_option"
```

Reply:
https://github.com/trocco-io/terraform-provider-trocco/pull/38/commits/df37ac26ab335ee08923e16a6e2a471215ad39f5